src/views/partials/labels.handlebars

```diff
@@ -0,0 +1,22 @@
+<div>
+    <h3>Labels</h3>
+    <input type="checkbox" id="edit_labels">
+    <label for="edit_labels">Close</label>
+
+    {{!-- List Labels --}}
+    <form action="/boards/card/labels/{{boardId}}/{{cardId}}" method="PATCH" id="editLabels">
+        {{#each label}}
+            <input type="checkbox" checked id="card_label_{{@index}}">
+            <label for="card_label_{{@index}}"><p class="card-label-highlight" style="background-color: {{this.color}};"></p><p>{{this.text}}</p></label>
+        {{/each}}
+
+        {{!-- Add Label --}}
+        <form id="addLabel"> 
```

Comment:
this is still 2 forms. Only one form, get rid of this nested one inside of the larger form

Reply:
just use the input and label inside of this one

Reply:
Wouldn't that make the user have to reopen the page if they wanted to add multiple labels?

Reply:
Wouldn't it be better to have the add form add another checkbox with label name to the list then they can keep adding and removing if they want? If its easier to do the other way that's fine just thought this would be better IMO

Reply:
the way the route is set up is that it takes in the list of already existing labels, and a potential new label and updates the card to make sure it incorporates them all. For us to achieve MVP, lets do my structure and with time permitting next week, I am fine with looking into changing our code to reflect what you desire

Reply:
also unrelated to this, but when you merge please delete the branch haha

Reply:
i did it for cal-route for you, but in the future...

Reply:
Rip sorry, and ok now I gotcha :)